libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -482,8 +557,8 @@ export const AttributeFilterButtonCore: React.FC<IAttributeFilterButtonProps> =
 
     const { FilterError } = props;
 
-    return error || attributeError ? (
-        <FilterError error={error || attributeError} />
+    return elementsError || attributeError ? (
+        <FilterError error={stringify(elementsError) || stringify(attributeError)} />
```

Comment:
Why stringify? Should this be smething like `elementsError?.message ?? attributeError?.message ?? "Unknown error"`?